bugbug/models/component.py

```diff
@@ -203,3 +204,42 @@ def is_meaningful(product, component):
 
     def get_feature_names(self):
         return self.extraction_pipeline.named_steps["union"].get_feature_names()
+
+    def check(self):
+        success = super().check()
+
+        # Get the number of bugs per full component to fasten up the check
+        bugs_number = get_product_component_count()
+
+        # Check that the most meaningful product components stills have at
+        # least a bug in this component. If the check is failing that could
+        # means that:
+        # - A component has been renamed / removed
+        # - TODO: Complete this list
+
+        for product, component in self.meaningful_product_components:
+            full_comp = f"{product}::{component}"
+
+            if full_comp not in bugs_number.keys():
```

Comment:
We should also check it's > 0

Reply:
I have updated `get_product_component_count` docstring to says that we already do the > 0 check there.

Reply:
I'd make it return all components, even if they have 0 bugs, and then here check that 1) the product/component exist; 2) the product/component has more than 0 bugs (so we can have a more detailed error message).

Reply:
Sure, I will update the PR